setup/monitoring/stackdriver.md

```diff
@@ -31,22 +31,59 @@ Stackdriver, please contact us through the Spinnaker Slack channel.
 
 # Installing Stackdriver
 
-## Configuring the Microservices
+## Configure the Spinnaker Monitoring Daemon for Stackdriver
 
-If you are going to have the microservices push directly into stackdriver,
-then you need to set the following attributes to `services.stackdriver`
-in `spinnaker-local.yml`:
 
-Name | Type | Description
------|------|------------
-enabled | true or false | Whether or not to write to stackdriver directly
-projectName | Stackdriver Project name | The stackdriver project to write the metric data into. The default is the `providers.google.primaryCredentials.project` however this is is not necessarily correct. Especially if you are using Stackdriver but not using Google.
-credentialsPath | Path to json file | A path to the JSON file containing the service account credentials downloaded from the Google Developer's Console used to authenticate with Stackdriver. The default is providers.google.primaryCredentials.jsonPath, however this might not be the account or credentials you wish to use. If you leave the value empty, it will use the default Service Account credentials for the VM that you are deployed on.
+First, you need a service account with the `roles/monitoring.viewer` and
+`roles/monitoring.metricWriter` roles enabled. If you don't already have this,
+run the following commands:
 
-If you are on Google Cloud Platform and using the Default Service Account
-credentials then you may need to enable the Monitoring-Write scope.
-This scope is enabled by default, however depending on how you create
-the VMs that you are deploying into, the scope may have been left out.
+```bash
+SERVICE_ACCOUNT_NAME=spinnaker-monitoring-account
+SERVICE_ACCOUNT_DEST=~/.gcp/gce-monitoring-account.json
```

Comment:
That's right, splitting these out by function seems right for these docs - we can always write a codelab with a "super" service account that has all needed roles